config/example/game_options.txt

```diff
@@ -76,4 +76,4 @@ LAVALAND_BUDGET 60
 
 ## Cube monkey limit
 # Amount of cube monkeys that can be spawned before the game limits them
-CUBEMONKEY_CAP 32
\ No newline at end of file
+CUBEMONKEY_CAP 32
```

Comment:
Please revert the diff in here

Reply:
That's just adding a line ending, I think.